### DISCUSSION TITLE
How to connect Ender3 v2 screen to the SKR E3 V3?

### DISCUSSION THREAD

**Discussion by SuhzHome** [mriscoc/Ender3V2S1#333] [Q&A]
skr mini e3 v3 dont work with dwin lcd

can we fix this?

**Comment by DavidKrall88** [comment]
> 

You need change the original lcd cable to another one for skr mini e3 v3

This is the cable:
https://biqu.equipment/products/mini-e3-connecting-cable-for-ender-3-v2-display-screen?_pos=4&_psq=ender&_ss=e&_v=1.0

**Comment by BioCookieYT** [comment]
Hello,
You have to change the pinout of the display. Just rewire the cable like I did:
https://www.smith3d.com/skr-mini-e3-v2-v3-on-ender-3-v2-ender-3-s1-lcd-dwin-knob-screen
https://imgur.com/a/RiGhYAc

**Comment by mriscoc** [comment]
Please consolidate all discussions about BTT SKR here: https://github.com/mriscoc/Ender3V2S1/discussions/131
